src/windowsdesktop/src/bundle/bundle.wixproj

```diff
@@ -0,0 +1,165 @@
+    <AssemblyProduct>Microsoft Windows Desktop Runtime</AssemblyProduct>
+    <AssemblyCopyright>© Microsoft Corporation. All rights reserved.</AssemblyCopyright>
+    <AssemblyInformationalVersion>$(Version)</AssemblyInformationalVersion>
+    <AssemblyFileVersion>$(Version)</AssemblyFileVersion>
+    <AssemblyVersion>$(Version)</AssemblyVersion>
+    
+    <!-- WiX v5 PDB configuration - bundles don't publish PDBs -->
+    <WixToolsetCompilerPdbType>full</WixToolsetCompilerPdbType>
+    <PublishWindowsPdb>false</PublishWindowsPdb>
+    <NoWarn>$(NoWarn);WIX1098</NoWarn>
+
+    <!-- Globbing will automatically find any .wxl relative to the project file. Since we're simshipping localized content for
+         the bundle UI, we either need to exclude the .wxl files or place them in a directory outside of the project. For simshipping
+         we only need to ensure the files are pulled in as additional payloads into the UX container. -->
+    <DefaultItemExcludes>$(DefaultItemExcludes);LCID\**\*;theme\**\*</DefaultItemExcludes>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(PkgMicrosoft_DotNet_Build_Tasks_Installers)\build\wix5\bundle\upgradePolicies.wxs" />
+
+    <!-- Workload manifests will be included only if the source fragment was generated. -->
+    <Compile Include="$(WorkloadManifestsWxsPath)" Condition="'$(WorkloadManifestsWxsPath)' != '' AND Exists('$(WorkloadManifestsWxsPath)')" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.WixToolset.Bal.wixext" />
+    <PackageReference Include="Microsoft.WixToolset.Util.wixext" />
+    <PackageReference Include="Microsoft.WixToolset.UI.wixext" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <!-- Ensure the Windows Desktop Runtime is built before this bundle -->
+    <ProjectReference Include="..\sfx\Microsoft.WindowsDesktop.App.Runtime.sfxproj" />
+  </ItemGroup>
+
+  <!-- Collect VS insertion packages for .NET Core runtime components -->
+  <!-- Unified runtime component resolution following ASP.NET Core patterns -->
+  <!-- Simple MSI Resolution (ASP.NET Core Style) -->
+  <Target Name="GetBundledNETCoreRuntimeMsiFiles" BeforeTargets="SetInstallerInfo">
+    <PropertyGroup>
+      <_CurrentArch>$(RuntimeIdentifier.Replace('win-', ''))</_CurrentArch>
+    </PropertyGroup>
+
+    <!-- Look for runtime MSI files in typical locations (like ASP.NET Core does) -->
+    <ItemGroup>
+      <!-- Local build artifacts -->
+      <DotNetHostMsiCandidate Include="$(ArtifactsPackagesDir)Shipping\dotnet-host-*-win-$(TargetArchitecture).msi" />
+      <DotNetHostFxrMsiCandidate Include="$(ArtifactsPackagesDir)Shipping\dotnet-hostfxr-*-win-$(TargetArchitecture).msi" />
+      <DotNetRuntimeMsiCandidate Include="$(ArtifactsPackagesDir)Shipping\dotnet-runtime-*-win-$(TargetArchitecture).msi" />
+      
+      <!-- Transport packages -->
+      <DotNetHostMsiCandidate Include="$(NuGetPackageRoot)vs.redist.common.netcore.sharedhost.$(_CurrentArch).10.0\**\*.msi" />
```

Comment:
Is it okay to hardcode the version `10.0` ?

Reply:
good catch, updated.